dlt/__init__.py

```diff
@@ -32,6 +32,8 @@
 from dlt.pipeline import pipeline as _pipeline, run, attach, Pipeline, dbt, current as _current, mark as _mark
 from dlt.pipeline import progress
 
+
```

Comment:
too many empty lines. we actually have black in `devel` branch so those problems will be fixed automatically...